### PR TITLE
Adds pagination property to show page number on bottom of page

### PR DESCRIPTION
**Pull request by kentsalcedo**
Adds pagination property to show page number on bottom of page

Per the requested changes from the prior PR

- Moves conditional to render pagination to the render function
- uses class_root for className
- adds spacing between operator and number 
- adds a conditional to check for the direction to be row
- adds back padding which is smaller than before because it looked quite large after a background color was added and looked large compared to designs 

signed-off-by: Kent Salcedo <kentsalcedo@webmocha.com>

### CODE REVIEW

File: src/js/components/Article.js [hunk 1]
```diff
@@ -422,6 +424,19 @@ export default class Article extends Component {
     }
   }
 
+  _renderPagination () {
+    const childCount = React.Children.count(this.props.children);
```
Comment (alansouzati) [inline src/js/components/Article.js]: Minor suggestion: I believe this can be safely replaced by `Children.count`

**Comment by alansouzati** [comment]
If the page does not have a scroll the pagination is not showing at all. Is this the expected behavior? I believe this is caused by the `atBottom` state.

In mobile works ok because it is really likely to have a scroll in the browser. But for desktop it is weird.

**Comment by L0ZZI** [comment]
I'm seeing the pagination float over the Chapter Hook's footer causing overlapping text. I think we need to pull this back for v2. I'd like to see this get a little more refinement before we push it in with a day left.

**Comment by L0ZZI** [comment]
Also if we do end up adding it in v1, let's not add the background to the navigation arrows, as i think that needs a little more animation and subtlety to hit the levels the Marketing folks want.

**Comment by kentsalcedo** [comment]
Per the new insights, I am closing this PR and will resubmit once it is more refined

**Pull request closed without merging**